uk.co.compucorp.civicrm.tasksassignments/views/modal/assignment.html

```diff
@@ -1,250 +1,332 @@
-<form novalidate name="assignmentForm" class="form-assignment form-horizontal" role="form" ng-submit="confirm()" ct-spinner>
+<form novalidate name="assignmentForm" class="form-assignment form-horizontal modal-inverted" role="form"
+      ng-submit="confirm()"
+      ct-spinner>
     <div class="modal-header">
-        <button type="button" class="close" ng-click="cancel()"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
-        <h4 class="modal-title" ng-bind="cache.contact.obj[assignment.contact_id].sort_name || 'No target contact'"></h4>
+        <button type="button" class="close" ng-click="cancel()"><span aria-hidden="true">&times;</span><span
+                class="sr-only">Close</span></button>
```

Comment:
*(Applies for any following instance)*

If you're gonna split the line on multiple lines, might as well then do the split consistent with the markup structure

```html
<button type="button" class="close" ng-click="cancel()">
    <span aria-hidden="true">&times;</span>
    <span class="sr-only">Close</span>
</button>
```